recipes/libalsa/all/conanfile.py

```diff
@@ -30,6 +30,9 @@ def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("alsa-lib-{}".format(self.version), self._source_subfolder)
 
+    def build_requirements(self):
+        self.build_requires("libtool/2.4.6")
+
     def _configure_autotools(self):
         if not self._autotools:
             self._autotools = AutoToolsBuildEnvironment(self)
```

Comment:
I think adding the vars of `AutoToolsBuildEnvironment` is unnecessary.
autoreconf is about regeneration of the `configure` script +  `Makefile.in`  files.
They do not depend on the actual compiler.

What should be done, is add `run_environment=True` to the `self.run`'s in this block.